pkgs/top-level/all-packages.nix

```diff
@@ -281,6 +281,8 @@ in
   nixBufferBuilders = import ../build-support/emacs/buffer.nix { inherit (pkgs) lib writeText; inherit (emacsPackagesNg) inherit-local; };
 
   pathsFromGraph = ../build-support/kernel/paths-from-graph.pl;
+  
+  singularityTools = callPackage ../build-support/singularity { };
```

Comment:
Might be good to rename to `build-support/singularity-tools`, so attribute path matches the file name.